packages/components/textarea/index.js

```diff
@@ -132,13 +132,12 @@ const Textarea = ({
             {numerationValue.join("\n")}
           </Numeration>
         )}
-
         <StyledTextarea
           id={id}
           paddingLeftProp={paddingLeftProp}
           isJSONField={isJSONField}
           enableCopy={enableCopy}
-          placeholder={placeholder}
+          placeholder={placeholder || " "}
```

Comment:
Why not fix the value in the default props?

Reply:
Moved it to default props